src/main/java/com/ylysenkova/movieland/dao/mapper/UserMapper.java

```diff
@@ -15,7 +15,7 @@ public User mapRow(ResultSet resultSet, int i) throws SQLException {
         user.setId(resultSet.getInt("id"));
         user.setName(resultSet.getString("user_name"));
         user.setEmail(resultSet.getString("user_email"));
-        user.setNickname(resultSet.getString("nickname"));
+        user.setPassword(resultSet.getString("nickname"));
```

Comment:
1. set pass, but from nickname;
2. do you really need to store user's pass? why?